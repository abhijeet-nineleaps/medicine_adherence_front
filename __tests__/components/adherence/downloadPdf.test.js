import Enzyme, { shallow } from "enzyme";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";
import React from "react";
import toJson from "enzyme-to-json";
import DownloadPdf from "../../../src/components/adherence/downloadPdf";

Enzyme.configure({ adapter: new Adapter() });

jest.mock("rn-fetch-blob", () => ({
  default: jest.fn(),
}));
describe("test collector category", () => {
  it("test category", () => {
    let globalmedId = 6;
    const wrapper = shallow(<DownloadPdf {...globalmedId} />);
    expect.assertions(1);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});