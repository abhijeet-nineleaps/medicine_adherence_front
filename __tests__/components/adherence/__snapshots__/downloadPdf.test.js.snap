// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test collector category test category 1`] = `<Component />`;
